### PR TITLE
PMM-6983 Add force for unregister k8s

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]
looks good to me, api descriptor from my PR?

**Comment by JiriCtvrtka** [comment]
@BupycHuk As far as I remember I created this branch from release/2.13, maybe this is reason?

**Review by AlekSi — CHANGES_REQUESTED** [review]
This field has to be documented

File: api/managementpb/dbaas/kubernetes.proto [hunk 1]
```diff
@@ -56,6 +56,9 @@ message UnregisterKubernetesClusterRequest {
       string_not_empty: true
     }
   ];
+  // If true then Kubernetes cluster will be deleted
+  // even if it contains clusters.
```
Comment (AlekSi) [inline api/managementpb/dbaas/kubernetes.proto:60]:
```suggestion
  // even if it contains database clusters.
```

Comment (JiriCtvrtka) [reply api/managementpb/dbaas/kubernetes.proto:60]: Sounds better. I will changed it.

**Comment by JiriCtvrtka** [comment]
FB: https://github.com/Percona-Lab/pmm-submodules/pull/1273

**Pull request merged**